src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -4959,6 +5293,18 @@ protected override PSProperty DoGetProperty(object obj, string propertyName)
             return new PSProperty(nodes[0].LocalName, this, obj, nodes);
         }
 
+
+        private protected override PSProperty DoGetFirstPropertyOrDefault(object obj, MemberNamePredicate predicate)
+        {
+            XmlNode node = FindFirstNodeOrDefault(obj, predicate);
+            if (node == null)
```

Comment:
```suggestion
            return (node == null) ? null : new PSProperty(node.LocalName, this, obj, node);
```
Replace the below lines also, if you take the suggestion.

Reply:
Operator `??`

Reply:
Operator ?? is the inversion.